packages/expo-image/src/web/style.tsx

```diff
@@ -0,0 +1,121 @@
+          opacity: 0;
+        }
+        .flip-from-right {
+          transition-property: transform, opacity;
+          transition-timing-function: var(--expo-image-timing,linear), steps(2, jump-none) !important;
+          transform-origin: center;
+        }
+        .flip-from-right-container {
+          width: 100%;
+          height: 100%;
+          position: absolute;
+          left: 0;
+          top: 0;
+          perspective: 1000px;
+        }
+        .flip-from-right-start:not(.transitioning) {
+          transform:  translateZ(calc(var(--expo-image-width,1000px) * -1.25)) rotateY(180deg);
+          opacity: 0;
+        }
+        .flip-from-right-active {
+          transform: translateZ(0px) rotateY(0) ;
+          opacity:1;
+        }
+        .flip-from-right-end {
+          transform:  translateZ(calc(var(--expo-image-width,1000px) * -1.25)) rotateY(-180deg);
+          opacity: 0;
+        }
+        .flip-from-top {
+          transition-property: transform, opacity;
+          transition-timing-function: var(--expo-image-timing,linear), steps(2, jump-none) !important;
+          transform-origin: center;
+        }
+        .flip-from-top-container {
+          width: 100%;
+          height: 100%;
+          position: absolute;
+          left: 0;
+          top: 0;
+          perspective: 1000px;
+        }
+        .flip-from-top-start:not(.transitioning) {
+          transform:  translateZ(calc(var(--expo-image-height,1000px) * -1.5)) rotateX(180deg);
+          opacity: 0;
+        }
+        .flip-from-top-active {
+          transform: translateZ(0px) rotateX(0) ;
+          opacity:1;
+        }
+        .flip-from-top-end {
+          transform:  translateZ(calc(var(--expo-image-height,1000px) * -1.5)) rotateX(-180deg);
+          opacity: 0;
+        }
+        .flip-from-bottom {
+          transition-property: transform, opacity;
+          transition-timing-function: var(--expo-image-timing,linear), steps(2, jump-none) !important;
+          transform-origin: center;
+        }
+        .flip-from-bottom-container {
+          width: 100%;
+          height: 100%;
+          position: absolute;
+          left: 0;
+          top: 0;
+          perspective: 1000px;
+        }
+        .flip-from-bottom-start:not(.transitioning) {
+          transform:  translateZ(calc(var(--expo-image-height,1000px) * -1.25)) rotateX(-180deg);
+          opacity: 0;
+        }
+        .flip-from-bottom-active {
+          transform: translateZ(0px) rotateX(0) ;
+          opacity:1;
+        }
+        .flip-from-bottom-end {
+          transform:  translateZ(calc(var(--expo-image-height,1000px) * -1.25)) rotateX(180deg);
+          opacity: 0;
+        }
+          `;
+export default function loadStyle() {
+  const styleTag = document.createElement('style');
+  styleTag.innerHTML = style;
+  document.head.appendChild(styleTag);
```

Comment:
So we cannot scope these styles and put inside the image container? The problem here is that these class names are a bit generic and the user may already have some elements that would be affected by these styles. Maybe at least prefix them with `expo` or `expo-image`?

Reply:
Yes, added a descendant selector